### PR TITLE
Added more info to newfile.

### PR DESCRIPTION
**Pull request by ArseniyKD**
This is a new update to newfile.

### CODE REVIEW

File: newfile.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 this is definitely not a new file.
+
+also, this was a new file.
```
Comment (kuzi117) [inline newfile.txt:3]: No, you're wrong, this isn't, see https://github.com/kuzi117/test/pull/3/files#diff-5b636c7ec81a3f0133aa1c887e09298dR1

File: newfile.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 this is definitely not a new file.
+
```
Comment (kuzi117) [inline newfile.txt:2]: This is not the correct style, please remove new lines.

**Review by kuzi117 — CHANGES_REQUESTED** [review]
This isn't ARseniy, you should know better.

File: newfile.txt (hunk 1 above)
Comment (ArseniyKD) [reply newfile.txt]: You are totally right!

Comment (kuzi117) [reply newfile.txt]: I've fixed this db8c8d9258d4318e393d0a7b33700f46e34ef1df

File: newfile.txt (hunk 2 above)
Comment (ArseniyKD) [reply newfile.txt]: Done.

**Review by kuzi117 — APPROVED** [review]
This is better, thank you.

**Pull request merged**